xrootd.sh

```diff
@@ -1,7 +1,7 @@
 package: XRootD
-version: "%(tag_basename)s"
-tag: v3.3.6-alice2
-source: https://github.com/alisw/xrootd.git
+version: "%(short_hash)s"
```

Comment:
You should override `version:`, `tag:`, and `source:` in the `defaults-jalien.sh` file, not here.

Reply:
I believe I have updated the xrootd recipe to use the official xrootd repo,
hence the new values of tag, version and source. I then override these
in the jalien defaults for @costing particular case.